zipline/data/us_equity_pricing.py

```diff
@@ -382,6 +391,16 @@ def __init__(self, table):
             int(id_): offset
             for id_, offset in iteritems(table.attrs['calendar_offset'])
         }
+
+        if "first_trading_day" in table.attrs.attrs:
```

Comment:
Both `table.attrs.attrs` and `table.attrs` are referenced, is that intended?

Usually in this 'if available use the dict value' paradigm, we use KeyError, e.g. 

```
try:
    self._first_trading_day = Timestamp(tables.attrs['first_trading_day'], unit='ms', tz='UTC'))
except KeyError:
    self._first_trading_day = None